internal/server/routes.go

```diff
@@ -40,16 +40,13 @@ func (s *Server) registerRoutes() *mux.Router {
 
 	r.Handle("/events/{eventKey}/teams", s.eventTeamsHandler()).Methods(http.MethodGet)
 	r.Handle("/events/{eventKey}/teams/{teamKey}", s.eventTeamHandler()).Methods(http.MethodGet)
-	r.Handle("/events/{eventKey}/teams/{teamKey}/comments", ihttp.ACL(s.getEventComments(), false, false, false)).Methods(http.MethodGet)
+	r.Handle("/events/{eventKey}/teams/{teamKey}/comments", ihttp.ACL(s.getEventTeamComments(), false, false, false)).Methods(http.MethodGet)
```

Comment:
Do you think it might make sense to have a event team reports endpoint since we're now storing comments as part of reports? It seems weird that a PUT to `/events/{eventKey}/matches/{matchKey}/reports/{teamKey}` will add something to `/events/{eventKey}/teams/{teamKey}/comments` I guess. We could also maybe add a query parameter that controls whether we send back report data in case they don't care about that (they probably will never care about it)

Reply:
Good idea, @calebeby do you want a query param or nah?

Reply:
I don't think we need a query parameter, or this endpoint. Let's just have it send the reports with the matches, always